app/src/main/res/layout/item_categories.xml

```diff
@@ -7,9 +7,29 @@
 
     <TextView
         android:id="@+id/categorie_text"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
+        android:layout_width="166dp"
+        android:layout_height="35dp"
```

Comment:
Just a comment: 

Use caution when specific width.  If you change the language/text you can destroy your view.